src/renderer/components/+apps-releases/releases.tsx

```diff
@@ -138,7 +138,7 @@ export class HelmReleases extends Component<Props> {
             },
             ...headerPlaceholders,
           })}
-          renderHeaderTitle="Releases"
+          renderHeaderTitle="Helm Releases"
```

Comment:
This change is not related to Electron version bump(?)

Reply:
No and I will revert